handsontable/src/plugins/autofill/autofill.js

```diff
@@ -15,6 +15,8 @@ const SETTING_KEYS = ['fillHandle'];
 const INSERT_ROW_ALTER_ACTION_NAME = 'insert_row';
 const INTERVAL_FOR_ADDING_ROW = 200;
 
+/* eslint-disable jsdoc/require-description-complete-sentence */
```

Comment:
It would probably be a better idea to disable this eslint rule just for the problematic section. See the comment below.

Reply:
The `complete-sentence` rule is not useful and causes legibility problems. Can we just turn it off for all JSDocs?